src/platform-integration/web/faq.md

```diff
@@ -110,6 +110,55 @@ with the location of your hosted HTML page:
 <iframe src="URL"></iframe>
 ```
 
+Another way of embedding is In your HTML file where you want to embed the Flutter app,
+create a container using a <div> element. Give it an id so that you can reference it in
+JavaScript. 
+
+```html
+<div id="flutter-container"></div>
+```
+In the same HTML file, include the JavaScript files generated by Flutter in the <body> section. 
+Add the following scripts:
+```html
+<script src="path/to/flutter.js"></script>
+<script src="path/to/flutter_assets/main.dart.js"></script>
+```
+Replace "path/to/flutter.js" and "path/to/flutter_assets/main.dart.js" with the actual paths to your
+Flutter-generated JavaScript files.
```

Comment:
**chore:** Code blocks require one empty line before and after. 
Write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion

   ```html
   <div id="flutter-container"></div>
   ```

1. In the `<body>` section,
   add `script` tags linking to the Flutter-generated JavaScript files. 

   ```html
   <script src="path/to/flutter.js"></script>
   <script src="path/to/flutter_assets/main.dart.js"></script>
   ```

   Replace "path/to/flutter.js" and "path/to/flutter_assets/main.dart.js"
   with the paths to your Flutter-generated JavaScript files.
```